crates/data_components/src/cdc.rs

```diff
@@ -331,6 +331,38 @@ pub fn wrap_data_as_change_batch(
     ChangeBatch::try_new(record_batch)
 }
 
+pub fn replace_change_batch_data(
```

Comment:
Moved as-is from crates/data_components/src/debezium/arrow/changes.rs

This logic is not debezium-specific and we can keep it as general CDC functionality